solarforecastarbiter/metrics/preprocessing.py

```diff
@@ -58,6 +58,84 @@ def apply_validation(obs_df, qfilter, handle_func):
         return validated_obs, counts
 
 
+def _resample_event_obs(obs, fx, obs_series):
+    """
+    Resample the event observation.
+
+    Parameters
+    ----------
+    obs : datamodel.Observation
+        The Observation being resampled.
+    fx : datamodel.EventForecast
+        The corresponding Forecast.
+    obs_series : pd.Series
+        Timeseries data of the event observation.
+
+    Returns
+    -------
+    obs_resampled : pandas.Series
+        Timeseries data of the Observation resampled to match the Forecast.
+
+    Raises
+    ------
+    RuntimeError
+        If the Forecast and Observation do not have the same interval length.
+
+    """
+
+    if fx.interval_length != obs.interval_length:
+        raise RuntimeError("Event observation and forecast time-series "
```

Comment:
I thought `ValueError` would be more appropriate, but I don't know for sure.

Reply:
Agreed; I'll revise to `ValueError`.